### ISSUE TITLE
Dismissible Health Checks

### ISSUE THREAD

**Issue by carlos-443-diaz** [Radarr/Radarr#7570]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe

Health messages can not be silenced, paused, hidden, etc.  This is mostly a concern for {titles} "were removed from TMDb", and "Enable Completed Download Handling".

### Describe the solution you'd like

I'd like to be able to snooze or dismiss messages in the Health section.

### Describe alternatives you've considered

- Not watching for movies that are too far out, and may not get a production green-light.  I don't have a readily available means to make this determination.
- Enabling completed download handling.  This doesn't work for me as I have a conversion workflow that is automated, but not orchestrated that is executed between download and adding to my media library.

### Anything else?

Thank you for your consideration and all the hard work that has gone into making such a well-rounded product.

[AB#3683](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/3683)

**Comment by bakerboy448** [comment]
> Enabling completed download handling. This doesn't work for me as I have a conversion workflow that is automated, but not orchestrated that is executed between download and adding to my media library.

There's likely no reason this workflow can't be modified to support radarr.  Either convert after radarr imports - which can be triggered with a custom script - or get creative and have your download client trigger the workflow and ensure the file remains completely hidden from radarr until it's converted, likely with some remote path map usage.

> This is mostly a concern for {titles} "were removed from TMDb"

This is a valid issue and likely would not be dismissible. Movies removed from tmdb will not receive any metadata updates which for some movies means they will never be available.  It is illogical to keep movies that do not exist and won't receive any metadata updates in radarr.

partial duplicate of https://github.com/Radarr/Radarr/issues/5393

**Comment by n0i-dev** [comment]
Hi! Checking in to see if this is being actively worked on; was interested in picking this Issue up if nobody else is on it.

**Comment by bakerboy448** [comment]
Go for it, but upstream in [Sonarr](https://github.com/Sonarr/Sonarr)and then pulled downstream

**Comment by harkus** [comment]
I have movies that were previously listed in TMDb but have since been removed. I still want to keep and track them in Radarr, just as before. Currently, the health check messages keep showing for these titles, and there’s no way to dismiss or acknowledge them.

While this may seem illogical to some, there are valid use cases for keeping these movies tracked without constant warnings. Please consider adding a “dismiss” or “acknowledge” option for this health check.

**Comment by brad-charboneau** [comment]
I agree, dismissible Health Alerts would be nice, I have multiple movies myself that show as an issue, instead of just Info since they were removed from TMDB.  Would be nice to have it not show as an issue at the very least.

**Comment by bakerboy448** [comment]
A simple "+1" in response to the original post would be sufficient for users who are in favor of this.

However, comments that do not contribute constructively to the discussion, such as "+1," "me too," "I want this too," "When will this be implemented?" or "Any updates?" do not enhance the dialogue and do not expedite the resolution of the issue. Repeated comments of this nature may lead to the thread being locked.

**Comment by zemerdon** [comment]
@bakerboy448 hey mate did you end up following through with this ? this is such a valid feature.  TMdB removing stuff for what they consider doesn't meet their rules but nontheless is still a valid piece media to ones library.

Would be perfect to be able to just dismiss said notification.  If we wernt tied to TMdB there would be options ;/

Would be nice to be able to create your own local metadata database and have radarr/sonarr resolve from that xD
